packages/constants/src/lib/constants/constants.ts

```diff
@@ -471,83 +510,39 @@ export const LIT_CHAINS: LITChain<LITEVMChain> = {
    * Chainlist entry for the Chronicle Testnet.
    * https://chainlist.org/chain/175177
    */
-  chronicleTestnet: {
-    contractAddress: null,
-    chainId: 175177,
-    name: 'Chronicle - Lit Protocol Testnet',
-    symbol: 'tstLIT',
-    decimals: 18,
-    rpcUrls: [
-      'https://lit-protocol.calderachain.xyz/replica-http',
-      'https://chain-rpc.litprotocol.com/http',
-    ],
-    blockExplorerUrls: ['https://chain.litprotocol.com/'],
-    type: null,
-    vmType: 'EVM',
-  },
+  chronicleTestnet: oldChronicleChain,
 
   /**
-   * @deprecated Will be removed in version 7.x. - Use `chronicleVesuviusTestnet` instead.
+   * @deprecated Will be removed in version 7.x. - Use `vesuvius` instead.
    */
-  datilDevnet: {
-    contractAddress: null,
-    chainId: 2311,
-    name: 'Chronicle Vesuvius - Lit Protocol Testnet',
-    symbol: 'tstLPX',
-    decimals: 18,
-    rpcUrls: ['https://vesuvius-rpc.litprotocol.com/'],
-    blockExplorerUrls: ['https://vesuvius-explorer.litprotocol.com/'],
-    type: null,
-    vmType: 'EVM',
-  },
+  datilDevnet: vesuviusChain,
+
+  /**
+   * @deprecated Will be removed in version 7.x. - Use `vesuvius` instead.
+   */
+  chronicleVesuviusTestnet: vesuviusChain,
 
   /**
    * Use this for `DatilDev` network.
    * Chainlist entry for the Chronicle Vesuvius Testnet.
    * https://chainlist.org/chain/2311
    */
-  chronicleVesuviusTestnet: {
-    contractAddress: null,
-    chainId: 2311,
-    name: 'Chronicle Vesuvius - Lit Protocol Testnet',
-    symbol: 'tstLPX',
-    decimals: 18,
-    rpcUrls: ['https://vesuvius-rpc.litprotocol.com/'],
-    blockExplorerUrls: ['https://vesuvius-explorer.litprotocol.com/'],
-    type: null,
-    vmType: 'EVM',
-  },
+  vesuvius: vesuviusChain,
+
+  /**
+   * @deprecated Will be removed in version 7.x. - Use `yellowstone` instead.
+   */
+  chronicleYellowstoneTestnet: yellowstoneChain,
```

Comment:
May as well :hocho: this entire entry since it was never merged into the shipped SDK, no?